_includes/search.html

```diff
@@ -0,0 +1,5 @@
+<input placeholder="Search…" type="search" id="search" class="grid-block shrink search">
+<div class="popover popover-search-wrapper">
+  <div role="dialog" class="grid-block vertical popover-search lunr-hidden" id="results">
+  </div>
```

Comment:
Since this element is empty, this should be on the same line as the open `<div>`

For example:
`<div></div>` 

As opposed to:
```
<div>
</div>
```